common/src/main/java/org/apache/celeborn/common/protocol/message/StatusCode.java

```diff
@@ -77,7 +77,8 @@ public enum StatusCode {
   PUSH_DATA_MASTER_BLACKLISTED(44),
   PUSH_DATA_SLAVE_BLACKLISTED(45),
 
-  FETCH_DATA_TIMEOUT(46);
+  FETCH_DATA_TIMEOUT(46),
+  REVIVE_INITIALIZED(47);
```

Comment:
I am not sure if we should add this here, maybe better a separated enum?

Reply:
I think it's fine, it is a status.

Reply:
> I think it's fine, it is a status.

But it's more like below 
```
object CommitInfo {
  val COMMIT_NOTSTARTED: Int = 0
  val COMMIT_INPROCESS: Int = 1
  val COMMIT_FINISHED: Int = 2
}

```